content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1007 @@
+
+* @mockersf now has merge rights for "uncontroversial changes"
+* @alice-i-cecile now has merge rights for "uncontroversial documentation changes"
+
+### New issue labels
+
+After [much discussion about naming conventions and colors](https://github.com/bevyengine/bevy/issues/2256), we finally have a [fresh new set of issue labels](https://github.com/bevyengine/bevy/labels) (loosely inspired by the rust repo). The [Bevy Triage Team](https://github.com/orgs/bevyengine/teams/triage-team/members) can finally express themselves fully!
+
+### Comprehensive CONTRIBUTING.md
+
+<div class="release-feature-authors">authors: @alice-i-cecile</div>
+
+We now have a relatively complete [Contributors Guide](https://github.com/bevyengine/bevy/blob/main/CONTRIBUTING.md). If you are interested in contributing code or documentation to Bevy, that is a great place to start! 
+
+### CI Build System Improvements
+
+<div class="release-feature-authors">authors: @mockersf, @NathanSWard, @NiklasEi</div>
+
+We made a ton of CI improvements this release:
+
+* We now fail on cargo doc warnings
+* We now use [cargo deny](https://github.com/EmbarkStudios/cargo-deny) to protect against vulnerabilities, duplicate dependencies, and invalid licenses
+* PRs are now automatically labeled with the `S-Needs-Triage` label
+* Ci stability and speed improvements
+* We now check that our benchmarks build
+* We now assert compiler errors for compile_fail tests, giving us much stricter guarantees
+* Examples are now run using lavapipe (instead of swiftshader) for faster CI validation
+
+## What's Next For Bevy?
+
+Bevy development continues to pick up steam and we have no intention to slow down now!
+
+### The "Train" Release Schedule
+
+In the last two Bevy releases we made massive, sweeping changes to core systems. **Bevy 0.5** was "the one where we rewrote Bevy ECS". **Bevy 0.6** was "the one where we rewrote Bevy Render". These massive reworks took time, and as a result held back a bunch of other useful features and bug fixes.
+
+[The Bevy Community](https://bevyengine.org/community/) has reached relative consensus that we should have a more regular, more predictable release schedule. One where large features can't gum up the system.
+
+From now on, we will cut releases _approximately_ once every three months (as an upper bound ... sometimes we might release early if it makes sense). After the end of a release cycle, we will start preparing to cut a release. If there are small tweaks that need to be made or "life happens" ... we will happily postpone releases. But we won't hold releases back for "big ticket" items any more.
```

Comment:
Ideally, I'd like us to go as far as to have fixed predetermined release dates, that can only be changed in an emergency. Say, something like: the first Thursday of {January, April, July, October}. Even put the date of the next release in the main Readme on github, so everyone can plan around it.

I know I might be asking for too much. I don't insist. Just stating a proposal for what I think would be ideal, given all the concerns you listed.

Ultimately, it's up to you, @cart, to decide. Do what feels right. :)

Reply:
Disclaimer: I've thought a lot about this one and I'm pretty resolved at this point.

The reasons to avoid absolute / fixed dates are as follows:
* If something is broken / release blocking, we shouldn't feel pressure to crunch just to meet an arbitrary deadline
* We shouldn't arbitrarily limit ourselves if something important is just a day or so from being ready

The goal of the "train release schedule" is to ensure we don't delay releases and establish some level of "trust" in the system.

We can accomplish that by "kicking off" releases on a regular schedule and generally trying to release "on time", but with just a small amount of wiggle room to account for human / life factors. Ideally we hit our targets every time (and I suspect that we will get better at it over time). But our users don't actually care about exact 3-month cadences, so we shouldn't stress about it either.

Reply:
Okay! I'm happy with that. Thanks for clarifying your final thoughts on the subject and how you envision it panning out.